playground/mocks/data/idp/idx/authenticator-enroll-phone.json

```diff
@@ -40,15 +40,16 @@
                         "value": "sms"
                       },
                       {
-                        "label": "VOICE",
+                        "label": "Voice call",
                         "value": "voice"
                       }
                     ]
                   },
                   {
                     "name": "phoneNumber",
                     "required": true,
```

Comment:
Looks like `required` field is set to `false` [here](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/867816386/End+User+Authenticator+Enrollment+Verification+Remediation#Example), hence added the change to [OKTA-304888](https://oktainc.atlassian.net/browse/OKTA-304448)